resdk/resolwe.py

```diff
@@ -494,7 +497,7 @@ def __init__(self, resolwe, Resource, endpoint=None):
         self.resolwe = resolwe
         self.resource = Resource
         self.endpoint = endpoint if endpoint else Resource.endpoint
-        self.api = getattr(resolwe.api, self.endpoint)
+        self.api = operator.attrgetter(self.endpoint)(resolwe.api)
```

Comment:
It can resolve nested paths like `kb.feature.admin`, while `getattr` would require three nested calls to achieve the same (eg. `getattr(getattr(getattr(x, 'kb'), 'feature'), 'admin')`).